### ISSUE TITLE
Multi-node mining, wallet addresses

### ISSUE THREAD

**Issue by f8vze** [RadiantBlockchain/rad-bfgminer#14]
Two questions.

1. When using this command:
```
/root/rad-bfgminer/bfgminer -S opencl:auto -o http://master.radiantblockchain.org:7332 -u raduser -p radpass --set-device OCL:kernel=poclbm --coinbase-sig somesig123 --generate-to someaddress
```
From various different machines (different IP/hardware etc.), will the `somesig123` `coinbase-sig` ensure that each miner is working on unique work?

2. If I understand correctly, Radiant supports bitcoint-like wallet addresses, and each bitcoin address is also a valid Radiant wallet address?

Thank you

**Comment by bkn7** [comment]
1. Yes, if you use a different sig or address per machine
2. Yes

**Comment by f8vze** [comment]
Thank you for responding.

On 2: great.

On 1: @z7k6 when you say "if you use a different sig or address per machine" do you mean that if you use the same coinbase-sig and the same payment address it will be doing double work? Also, if the coinbase-sig and address are both different, how can machines "work together" on the current block? Or is the answer to use only one different item (i.e. either coinbase-sig or address is unique, but not the other?)

Basically I am trying let multiple machines work together, avoiding indentical work on the same block being mined.

**Comment by bkn7** [comment]
If multiple machines use the same signature and address they would be duplicating work. The signature or address should be different per machine. If you need more help with mining the mining channel on Discord is very active.

**Comment by f8vze** [comment]
Thank you. So to clarify, **both** need to be different? How then can they syncronize on work done? i.e. how does miner instance 1 ensure it is not trying to do the same part of the block as miner 2?

**Comment by bkn7** [comment]
At least one of them needs to be different. The miners do not need to be synchronised, they will be doing different work if the signature or address is different.

**Comment by f8vze** [comment]
@z7k6 That is very interesting. Is that because it is being pointed to `master.radiantblockchain.org` - i.e. does that act as a pool which allocates work? For solo mining, a local server setup is required?
